src/server.ts

```diff
@@ -12,28 +12,31 @@ import { createContext, permissions } from './auth/index.js'
 import XMediaDataSource from './model/XMediaDataSource.js'
 import PostDataSource from './model/PostDataSource.js'
 import { createInstance as createNewOrgDS } from './model/MutableOrganizationDataSource.js'
+import type { Context } from './types.js'
+import type { DataSources } from 'apollo-server-core/dist/graphqlOptions'
 
 export async function createServer (): Promise<ApolloServer> {
   const schema = applyMiddleware(
     graphqlSchema,
     permissions.generate(graphqlSchema)
   )
+  const dataSources: () => DataSources<Context> = () => ({
```

Comment:
This is the key change which specifies the type instead of leaving it as `DataSources<object>`.